sdk/eventhub/event-hubs/README.md

```diff
@@ -371,21 +388,21 @@ export DEBUG=azure*
 export DEBUG=azure*,rhea*
```

Comment:
What happens if DEBUG is set to rhea* and AZURE_LOG_LEVEL is set to info
Do we get both? Or one takes precedence over the other?

Reply:
Based on my testing we do get both.

Reply:
I added a note: 

> **Note:** AZURE_LOG_LEVEL, if set, takes precedence over DEBUG.
Do not specify any `azure` libraries via DEBUG when also specifying
AZURE_LOG_LEVEL or calling setLogLevel.